lib/migrator/migrationSteps.js

```diff
@@ -1,28 +1,21 @@
 const path = require('path')
 const lodash = require('lodash')
 const fse = require('fs-extra')
-const { promises: fs } = require('fs')
 const { appDir, projectDir, packageDir } = require('../path-utils')
-const { getConfig } = require('../config')
-const { addReporter, reportSuccess, reportFailure } = require('./reporter')
```

Comment:
I don't think it is necessary to change this require, the tests pass for me without this change.